scipy/sparse/tests/test_base.py

```diff
@@ -3678,7 +3678,7 @@ def spcreator(cls, *args, **kwargs):
             sup.filter(SparseEfficiencyWarning,
                        "Changing the sparsity structure of a csr_matrix is expensive")
             return csr_matrix(*args, **kwargs)
-    math_dtypes = [np.bool_, np.int_, np.float64, np.complex128]
+    math_dtypes = [np.bool_, np_long, np.float64, np.complex128]
```

Comment:
I guess these are just tests so it doesn't matter, although the use looks more like "default integer" than long to me.
(Unless it lists all the C types or interfaces C code, that is really the meaning you want?)

Reply:
Addressed in https://github.com/scipy/scipy/pull/19394.